src/store.js

```diff
@@ -89,7 +89,7 @@ const store = new Vuex.Store({
         perms.allowSaveLocally = false;
       }
       // Disable saving changes to disk, only
-      if (appConfig.preventWriteToDisk || !isUserAdmin) {
+      if (appConfig.preventWriteToDisk || !isUserAdmin()) {
```

Comment:
This will **not** fix issue  #590 because if I understand correctly this is only a frontend check for admin privileges and therefore anyone can still realistically update the configuration file.
If you were visit the site as a guest, then proceed to add a "local" (not existing in conf.yml) user with admin role then proceed to log in into that user then you can still very much update the config file however you like. 
I'm suggesting a proper backend check for the already-known-good users/admins to permit unchecked file writes and command execution if there isn't one yet.

Reply:
Yeah, you're right, there really should be more backend checks.

I have explained the attack in more detail in the Auth docs, as this authentication method definitely shouldn't be used outside of your local network. Keycloak is supported for more secure authentication. But it should also be possible to protect the app with any self-hosted auth provider.

Potentially the best option, is to remove simple-auth from the app all together, and let users bring their own auth provider.

Reply:
I'm not really sure that removing the simple-auth would fix the issue, if there is no backend check. I mean it is nice that the password is handled by Keycloak for example, but if you don't need anything to activate a backend funtion, only the knowledge that where it is located and sending a direct request to it. 
I for one say that the password hash being publicly accessible a non-issue, but in my case Dashy is running on a mini-pc and there it is a problem if anyone can start a rebuild of the software or they can fill the available storage with freely written configuration files.
Of course what you are saying that this should be used within a local network is totally reasonable, I just advocate that we place approriate warnings, that this should not be publically accessible because of possible "attack vectors".